apps/expo/src/app/activity.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Text, View } from "react-native";
+import { AntDesign } from "@expo/vector-icons";
+
+// Props specific for the Activity component
+interface ActivityProps {
+  name: string;
+  location?: string;
+  attending?: boolean;
+  activityHeight?: number;
```

Comment:
should height be calculated from the duration of the activity?

Reply:
Yea. Thats what I was doing at first but thought that the height might be different based on situation (cause I saw on figma there were some that were varying sizes on different pages) so I was just gonna let the other screens decide. 

Another thing is that size is dependent on duration and line height. Since line height is decided in the schedule page, I didnt want to set it in both places and make it annoying to change in the future. 

Should I figure out a different way?

Reply:
If you think it's best to just add line height here, calculate it here, and have a book that dictates it came from schedule, then I can do that. Just let me know your thoughts!

Reply:
It might be better to just have this component fill the parent div, and control its size with the parent div instead?

Reply:
Ohhh interesting! I will do that! Thanks 🙏

Reply:
Fixed